### DISCUSSION TITLE
WordPress integration

### DISCUSSION THREAD

**Discussion by matt-sanders** [saadeghi/daisyui#2030] [Ideas / feature requests]
Hey 👋 
Love Daisy UI! I maintain [Gust](https://www.getgust.com/), which integrates Tailwind CSS with WordPress. I've been thinking about building in support for Daisy UI. Do you think this is something that people would find useful?

Technically users of Gust would already be able to do this just by adding the plugin to the Tailwind config, but I'd be adding all ( or most ) Daisy UI components to the drag and drop component library to make it easier for users.

To be clear, this doesn't really require anything from Daisy UI ( other than maybe adding a WP item on the [installation page](https://daisyui.com/docs/install/) ), I'm more just gauging interest.

**Comment by saadeghi** [comment]
That would be helpful. I think apart from the components, daisyUI color themes would be useful for WordPress users because they can change the colors of the page and all components easily.  

Let me know if I can help

**Comment by matt-sanders** [comment]
Awesome sounds good. I'll get onto it!

**Comment by matt-sanders** [comment]
@saadeghi I've recently finished up a plugin to add daisyUI support to WordPress via Gust. As I'm sure you've just seen, I've created a PR to add the WordPress installation details to the docs site.

The plugin adds a bunch of components into the page builder by default, as well as allowing the user to edit their theme and specify their default theme, as you mentioned this would be important.

If you want to give it a spin before approving, let me know and I can send you a production version of Gust so you can test it out.
